services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -1137,11 +1167,11 @@ protected File downloadFromUrlToNfs(String url, NfsTO nfs, String path, String n
             try (FileOutputStream outputStream = new FileOutputStream(destFile);) {
                 entity.writeTo(outputStream);
             } catch (IOException e) {
-                s_logger.debug("downloadFromUrlToNfs:Exception:" + e.getMessage(), e);
+                s_logger.error("downloadFromUrlToNfs:Exception:" + e.getMessage(), e);
             }
             return new File(destFile.getAbsolutePath());
         } catch (IOException e) {
-            s_logger.debug("Faild to get url:" + url + ", due to " + e.toString());
+            s_logger.error("Faild to get url:" + url + ", due to " + e.toString());
```

Comment:
Suggestion:

Pass the exception as parameter to `s_logger.error`:

```java
s_logger.error("Faild to get url:" + url + ", due to " + e.toString(), e);
```

Reply:
thanks will do it